catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -115,7 +123,8 @@ public String getWkt() throws CswException {
         // started
         reader.moveUp();
 
-        LOGGER.debug("Returning WKT in LON/LAT coord order: {}.", wkt);
+        wkt = convertToEPSG4326(wkt, crs);
```

Comment:
The underlying conversion will swap the coordinates, so I think we should have some check to not call it if it's already EPSG:4326 & lon/lat ordered.